### ISSUE TITLE
Data table content alignement in Create a relationship panels

### ISSUE THREAD

**Issue by Archidoit** [LuatixHQ/opencti#8937]
## Description
Data table content is not aligned with the columns titles in the Create a relationship panels

## Reproducible Steps
- Go to an entity
- Go to the Knowledge tab
- Create a relationship
- In the relationship panel, the content is not aligned with the column titles: 
![image](https://github.com/user-attachments/assets/b2e1a7d6-c86d-41c4-8901-3fbeaa82f026)

(see the labels /markings are too on the right compared to the title)

Content is aligned for other Data tables (exemple: reports list) : 
![image](https://github.com/user-attachments/assets/9e197095-3c87-499d-b132-4eb34c917152)

**Comment by labo-flg** [comment]
titles seem fixed, but we lost some top margin

![Image](https://github.com/user-attachments/assets/5e10b8ac-280e-47c7-80f5-1f2cf1e8d302)

**Comment by delemaf** [comment]
It's seems everything have been fixed, should we close this @nino-filigran ?
![Image](https://github.com/user-attachments/assets/591a18e2-8f02-4d51-976a-ad0727fbc73d)

**Comment by nino-filigran** [comment]
yes good for me, just checked on multiple entities & it seems fixed!
